content/docs/user-guide/integrations/sagemaker.md

```diff
@@ -0,0 +1,50 @@
+# Amazon SageMaker
+
+## Setup
+
+Many DVC features rely on Git. To work with DVC in Amazon SageMaker, first setup
+your Git repo:
+
+1. [Clone] a repository.
+
+   ![Git clone](/img/sagemaker-git-clone.gif)
+
+2. Launch a terminal or notebook and configure the user name and email:
+
+   ```bash
+   git config --global user.name ...
+   git config --global user.email ...
+   ```
+
+Don't forget to install DVC and any other requirements in your environment!
+
+## Notebooks
+
+After completing the setup, you can work with DVC in SageMaker notebooks like
+you would in any other environment. Take a look at DVC [experiments] for how to
+get started with DVC in notebooks. If you would like to see live experiment
+updates in [DVC Studio], set your [token].
```

Comment:
Suggestion

> If you would like to see live experiment updates in [DVC Studio], set your [token], for example:
> 
> ```bash
>    $ dvc config --global studio.token <MY_STUDIO_TOKEN>
> ``` 
> and you're ready to go!
> 


My reasoning is:
1. config command - for the most common Sagemaker scenarios, it would be nice to have everything here, without having to be sent to other pages for such things (link there for more info/options, etc...)
2. "and you're ready to go!" - I experienced a mental stutter going from the sentence about setting up the token to the gif showing experiment tracking (and NOT setting up the token)

Reply:
I ended up copying what we have in https://dvc.org/doc/user-guide/experiment-management/sharing-experiments#live-metrics-and-plots.